docs/topics/api/addons.rst

```diff
@@ -168,6 +168,7 @@ This endpoint allows you to fetch a specific add-on by id, slug or guid.
     :>json boolean public_stats: Boolean indicating whether the add-on stats are public or not.
     :>json object ratings: Object holding ratings summary information about the add-on.
     :>json int ratings.count: The number of user ratings for the add-on.
```

Comment:
Maybe change to `The total number of user ratings for the add-on` or something similar to contrast more with the one you added.